src/styles/index.scss

```diff
@@ -1,3 +1,90 @@
-body {
+* {
+  padding: 0;
   margin: 0;
+  box-sizing: border-box;
```

Comment:
Consider using a more specific selector instead of the universal selector '*' to avoid applying these styles to every single element on the page, which could lead to performance issues.